ingestion/src/metadata/data_insight/processor/reports/web_analytic_report_data_processor.py

```diff
@@ -117,9 +118,11 @@ def _refine_entity_event(self) -> Generator[dict, WebAnalyticEventData, None]:
                 # if we've seen the entity previously but were not able to get
                 # the URL we'll try again from the new event.
                 try:
-                    entity_href = re.search(
-                        re_pattern, event.eventData.fullUrl.__root__
-                    ).group(1)
+                    entity_href = unquote(
+                        re.search(
+                            re_pattern, quote(event.eventData.fullUrl.__root__)
+                        ).group(1)
+                    )
```

Comment:
We can revert back this to its original value once we changed the re pattern to escape re character